measurementWriter.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "fmt"
+
+type measurementWriter interface {
+	ReadMeasurement() error
+}
+
+type MeasurementPrintWriter struct {
+	mw          measurementWriter
+	measurement chan Measurement
+}
+
+func (mw *MeasurementPrintWriter) ReadMeasurement() error {
+	for {
+		fmt.Println("Id:", (<-mw.measurement).Id, "Value: ", (<-mw.measurement).Value)
```

Comment:
every `<-` blocks and waits for a message. In this line you are reading two different messages.